keyboards/neodox/keymaps/neodimio/keymap.c

```diff
@@ -0,0 +1,1227 @@
+}
+
+void suspend_wakeup_init_user(void) {
+    qp_power(lcd, true);
+    // rgb_matrix_set_suspend_state(false);
+    if (last_backlight != 255) {
+        backlight_set(last_backlight);
+    }
+    last_backlight = 255;
+}
+
+#ifdef OLED_ENABLE
+
+// // #ifdef OLED_ENABLE
+oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
+    return OLED_ROTATION_180;
+}
+
+
+
+void oled_write_wpm(void) {
+#ifdef WPM_ENABLE
+     // print WPM
+     oled_write_P(PSTR("\nWPM: "), false);
+     oled_write(get_u8_str(get_current_wpm(), ' '), false);
+ #endif
+ }
+
+bool oled_task_kb(void) {
+    if (!oled_task_user()) {
+        return false;
+    }
+    if (is_keyboard_master()) {
+        // QMK Logo and version information
+        // clang-format off
+        static const char PROGMEM qmk_logo[] = {
+            0x80,0x81,0x82,0x83,0x84,0x85,0x86,0x87,0x88,0x89,0x8a,0x8b,0x8c,0x8d,0x8e,0x8f,0x90,0x91,0x92,0x93,0x94,
+            0xa0,0xa1,0xa2,0xa3,0xa4,0xa5,0xa6,0xa7,0xa8,0xa9,0xaa,0xab,0xac,0xad,0xae,0xaf,0xb0,0xb1,0xb2,0xb3,0xb4,
+            0xc0,0xc1,0xc2,0xc3,0xc4,0xc5,0xc6,0xc7,0xc8,0xc9,0xca,0xcb,0xcc,0xcd,0xce,0xcf,0xd0,0xd1,0xd2,0xd3,0xd4,0};
+        // clang-format on
+
+        oled_write_P(qmk_logo, false);
+        oled_write_P(PSTR("Redox "), false);
+
+        // Host Keyboard Layer Status
+        oled_write_P(PSTR("Layer: "), false);
+        switch (get_highest_layer(layer_state | default_layer_state)) {
+            case _QWERTY:
+                oled_write_P(PSTR("QWERTY\n"), false);
+                break;
+            // case 1:
+            //     oled_write_P(PSTR("Dvorak\n"), false);
+            //     break;
+            // case 2:
+            //     oled_write_P(PSTR("Colemak-DH\n"), false);
+            //     break;
+            case _NAV:
+                oled_write_P(PSTR("Nav\n"), false);
+                break;
+            case _SYMB:
+                oled_write_P(PSTR("Sym\n"), false);
+                break;
+            case _FUNC:
+                oled_write_P(PSTR("Function\n"), false);
+                break;
+            case _MOUSE:
+                oled_write_P(PSTR("Mouse\n"), false);
+                break;
+            case _GAMING:
+                oled_write_P(PSTR("Gaming\n"), false);
+                break;
+            default:
+                oled_write_P(PSTR("Undefined\n"), false);
+
+        }
+        #ifdef WPM_ENABLE
+            oled_write_wpm();
+        #endif
+
+        oled_write_P(PSTR("\n"), false);
+
+        // Host Keyboard LED Status
+        led_t led_usb_state = host_keyboard_led_state();
+        oled_write_P(led_usb_state.num_lock ? PSTR("NUMLCK ") : PSTR("       "), false);
+        oled_write_P(led_usb_state.caps_lock ? PSTR("CAPLCK ") : PSTR("       "), false);
+        oled_write_P(led_usb_state.scroll_lock ? PSTR("SCRLCK ") : PSTR("       "), false);
+    } else {
+        // clang-format off
+        static const char PROGMEM redox_logo[] = {
+
+// 'Hylian , 128x64px
```

Comment:
While I personally love the logos... they're copyrighted content. And especially that Hylian logo (Nintendo is famously overzealously litigious).  

Please remove them.

Reply:
@drashna  thank you, I fixed all the issue